src/main/java/com/spotinst/sdkjava/model/ApiElastigroupListStatefulInstancesDevices.java

```diff
@@ -0,0 +1,78 @@
+package com.spotinst.sdkjava.model;
```

Comment:
General comment: Put all the files in the respective folder structure under "model" folder.

For eg: All Api class should go under model/api/elastigroup/aws, converter class should go under model/converters/elastigroup/aws